docs/api.rst

```diff
@@ -2361,6 +2361,12 @@ MessageInteractionMetadata
 .. autoclass:: MessageInteractionMetadata()
     :members:
 
+MessageCall
+~~~~~~~~~~~
+
+.. autoclass:: MessageCall()
```

Comment:
This is not imported in `__init__.py` so does not get found by docs, this should be imported in init.

Reply:
This was because I had not included MessageCall to message.py's dunder all variable, also fixed